src/lightning/data/streaming/writer.py

```diff
@@ -29,11 +29,35 @@
     from torch.utils._pytree import PyTree, tree_flatten, treespec_dumps
 
 
-def _get_data_optimizer_node_rank() -> Optional[int]:
-    node_rank = os.getenv("DATA_OPTIMIZER_NODE_RANK", None)
-    if node_rank is not None:
-        return int(node_rank)
-    return node_rank
+_FORMAT_TO_RATIO = {
+    "kb": 1024,
+    "mb": 1024**2,
+    "gb": 1024**3,
+    "tb": 1024**4,
+    "pb": 1024**5,
+    "eb": 1024**6,
+    "zb": 1024**7,
+    "yb": 1024**8,
```

Comment:
do you really think we'll get that far? :D

Reply:
Why not ;)